### DISCUSSION TITLE
Custom Battle Blog Link 404 Not Found

### DISCUSSION THREAD

**Discussion by SnowballSH** [cssbattle/forum#58] [Bugs reports]
In the "custom battle" page (https://cssbattle.dev/settings?tab=custom-battle-credits), after pressing 
![image](https://user-images.githubusercontent.com/66022611/119073202-739e3f00-b9b2-11eb-9eee-7839e79d4677.png), the browser opens up a new window (https://cssbattle.dev/blog/custom-battles) but showing 
![image](https://user-images.githubusercontent.com/66022611/119073236-81ec5b00-b9b2-11eb-89d6-43d3b40a428b.png)

Browser: Chrome

Please fix it so more people will be encouraged to buy credits :)

**Comment by chinchang** [comment]
Fixed now. thanks for reporting :)

**Reply by SnowballSH** [reply]
My pleasure :) Great website!
